.editorconfig

```diff
@@ -9,6 +9,6 @@ trim_trailing_whitespace = false
 insert_final_newline = true
 indent_style = tab
 
-[{*.yml,*.json}]
+[{*.yml,*.json,*.js}]
```

Comment:
all js files are indented with 2 spaces but editorconfig forces to indent with tab